README.md

```diff
@@ -50,3 +50,137 @@ Follow the steps below to run Export. To ensure .py files run when double-clicke
+* Meta can contain any info.
+  * This is a json object or python dictionary, one level deep.
+  * Need to use double quotes and separate each item by spaces i.e. {"firstname":"Patty","nickname":"Pat"}
+* analysis must match an id in Analysis.csv
+* Required fields: id, category, subcategory, name, icon, analysis
+
+## Requirements
+
+Same as Hazus Export.
+
+## To Use
+
+Follow the steps below to run Batch Export. To ensure .py files run when double-clicked, right-click the .py file and go to Properties. Under the "General" tab next to "Opens With", make sure "python.exe" is selected. If not, click "Change" and select "python.exe" from your Python installation directory.
+
+It's recommended that if you want to rerun an HPR that you should delete or rename the previous run's output folder for that HPR. 
+
+**1. Place your .hpr files into the batch_input folder**
+
+![batch input folder](Images/batchInputFolder.png "batch input folder")
+
+**1b. Adjust the desired outputs by modifying the values in the 'hazpy/batch_export.py'**
+Look near the very bottom of the script for these entries:
+
+    #USER DEFINED VALUES
+    hprDir = Path.absolute(Path(r'./batch_input'))      #The directory containing hpr files
+    outDir = Path.absolute(Path(r'./batch_output'))     #The directory for the output files
+    _outCsv = 1         #Export csv files: 1 to export or 0 to skip
+    _outShapefile = 1   #Export shapefiles: 1 to export or 0 to skip
+    _outReport = 1      #Export report pdf files: 1 to export or 0 to skip
+    _outJson = 1        #Export json files: 1 to export or 0 to skip
+
+If you wanted to export just the reports it would need to look like this:
+
+    #USER DEFINED VALUES
+    hprDir = Path.absolute(Path(r'./batch_input'))      #The directory containing hpr files
+    outDir = Path.absolute(Path(r'./batch_output'))     #The directory for the output files
+    _outCsv = 0         #Export csv files: 1 to export or 0 to skip
+    _outShapefile = 0   #Export shapefiles: 1 to export or 0 to skip
+    _outReport = 1      #Export report pdf files: 1 to export or 0 to skip
+    _outJson = 0        #Export json files: 1 to export or 0 to skip
+
+**2. Double-click 'batch-export-tool.py'. If you don't have the Hazus Python Library installed, follow the prompt to install, then double-click 'batch-export-tool.py' again**
+
+![batch-export-tool.py](Images/batchRun.png "batch-export-tool.py")
+
+**3. Check the 'batch_output' folder for the output**
+
+**4. For HLL, in the 'batch_output' folder replace "FIX ME" field values in the "Event.csv", "Analysis.csv" and "Downloads.csv" files**
+
+**Script Failure:**
+
+If the script fails without dropping the bk_* (where * is the name of the hpr file or its .bk sql server backup database) database then you will need to do it manually. Using SQL Server Management Studio you can DELETE the bk_* database and using Windows explorer you can delete the temp folder that contains the unzipped contents of the HPR file.
```

Comment:
add a comma before 'then' (see comment above).

**_If the script fails without dropping the bk_* **_(where * is the name of the hpr file or its .bk sql server backup database) database, then_**

Reply:
Good catch; change made.